gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -50,7 +50,8 @@ def actions
 
         actions = []
         actions << :clean if config[:clean]
-        actions << :archive
```

Comment:
It seems like we always want either `archive` or `build` here. If so then the if and unless blocks are wrong 😜